windows/pp_data_dump/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(pp_data_dump C)
+
+set(HIDAPI_pp_data_dump_TARGETS)
+if(WIN32)
```

Comment:
lets keep CMake here as simple as possible

remove the variable and the condition
none of that is needed - this target only included on Windows and will fail to build on non-windows platforms anyway